sdk/ml/azure-ai-ml/azure/ai/ml/_schema/_deployment/batch/batch_job_property.py

```diff
@@ -0,0 +1,75 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+# pylint: disable=unused-argument,no-self-use,protected-access
+
+from typing import Any
+
+from marshmallow import fields
+from marshmallow.decorators import post_load
+
+from azure.ai.ml._restclient.v2020_09_01_dataplanepreview.models import (
+    UriFileJobInput,
+    UriFolderJobInput,
+)
+from azure.ai.ml._schema.core.fields import ArmStr, NestedField
+from azure.ai.ml._schema.core.schema import PathAwareSchema
+from azure.ai.ml._schema.core.schema_meta import PatchedSchemaMeta
+from azure.ai.ml.constants import AssetTypes
+from azure.ai.ml.constants._common import AzureMLResourceType
+from azure.ai.ml.constants._endpoint import EndpointYamlFields
+
+from .batch_deployment_settings import BatchRetrySettingsSchema
+from .compute_binding import ComputeBindingSchema
+
+
+class OutputDataSchema(metaclass=PatchedSchemaMeta):
+    datastore_id = ArmStr(azureml_type=AzureMLResourceType.DATASTORE)
+    path = fields.Str()
+
+    @post_load
+    def make(self, data: Any, **kwargs: Any) -> Any:
+        from azure.ai.ml.entities._deployment.data_version import DataVersion
+
+        return DataVersion(**data)
+
+
+class BatchJobPropertySchema(PathAwareSchema):
```

Comment:
Multiple fields of BatchJobPropertySchema already exist in BatchJobSchema. Why the duplication? Can we get rid of the `properties` nesting of which I don't see much customer value?

Reply:
In the design it was agreed upon that we would expose all the values in batch job property at the level of batch job. However when invoke is called it returns a batch job property that we must use to create the batch job object that get returned to the cli. However, I can remove the properties attribute.

Reply:
I'm not following why batch job property has to be returned to cli?